pySPT/notebooks/diffractionLimit.py

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
Add docstring

"""
@author: Alexander Niedrig, Johanna Rahm
Research group Heilemann
Institute for Physical and Theoretical Chemistry, Goethe University Frankfurt a.M.

Python version of diffractionLimit.ipynb for batch processing analysis.
"""